plugins/variational_quantum_classifier/backend/visualization.py

```diff
@@ -138,24 +144,25 @@ def plot_data(
     predictor=None,
     only_first_100=True,
     title="",
+    label_to_int=None,
 ):
     # Prepare data
     dim = len(train_data[0])
     train_end = 100 if only_first_100 else len(train_data)
     test_end = 100 if only_first_100 else len(test_data)
 
-    train_data = np.array(train_data)[:train_end]
-    test_data = np.array(test_data)[:test_end]
+    train_data = np.array(train_data[:train_end])
+    test_data = np.array(test_data[:test_end])
     points = np.concatenate((train_data, test_data), axis=0)
 
     dimensions = ["x", "y", "z"][:dim]
     ids = (
         get_id_list(train_id_to_idx)[:train_end] + get_id_list(test_id_to_idx)[:test_end]
     )
 
-    train_labels = [str(int(el)) for el in train_labels[:train_end]]
-    test_labels = [str(int(el)) for el in test_labels[:test_end]]
-    labels = train_labels[:train_end] + test_labels[:test_end]
+    train_labels = [str(el) for el in train_labels[:train_end]]
+    test_labels = [str(el) for el in test_labels[:test_end]]
```

Comment:
The slices in this line and the one above should also not be required

Reply:
When creating a pandas.DataFrame all arrays must be of the same length, thus the slices are necessary.